src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -478,7 +478,18 @@ void InteractivityFactory::SetPseudoWindowCallback(std::function<void(bool)> fun
 // - <none>
 void InteractivityFactory::_WritePseudoWindowCallback(bool showOrHide)
 {
-    if (_pseudoWindowMessageCallback)
+    // BODGY
+    //
+    // GH#13158 - At least temporarily, only allow the PTY to HIDE the terminal
+    // window. There seem to be many issues with this so far, and the quickest
+    // route to mitigate them seems to be limiting the interaction here to
+    // allowing ConPTY to minimize the terminal only. This will still allow
+    // bodgy scripts to hide the Terminal via GetConsoleWindow(), but should

```

Comment:
Can you use ShowWindow to minimize it, then restore it, _and then use ShowWindow again in the same session_? Or does it get into a torn state?

Reply:
Sure does! This is only preventing `ShowWindow(GetConsoleWindow(), SW_SHOW)` from going out to the terminal, the rest of the state still flows _in_ to the pty window